tests/greenheart/test_hydrogen/input_files/plant/GS_greenheart_config.yaml

```diff
@@ -0,0 +1,194 @@
+  loan_period: 0 # H2FAST default, not used for revolving debt
+  cash_onhand_months: 1 # H2FAST default
+  administrative_expense_percent_of_sales: 0.00 #Check this # percent of sales H2FAST default
+  depreciation_method: "MACRS" # can be "MACRS" or "Straight line" - MACRS may be better and can reduce LCOH by more than $1/kg and is spec'd in the IRS MACRS schedule https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
+  depreciation_period: 5 # years - as specified by the IRS MACRS schedule https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
+  depreciation_period_electrolyzer: 7 # based on PEM Electrolysis H2A Production Case Study Documentation estimate of 7 years. also see https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
+  discount_years:
+    wind: 2022 # based on turbine capex value provided to ORBIT from 2022 ATB
+    wind_and_electrical: 2022 # for ORBIT opex
+    wave: 2020 # confirmed by Kaitlin Brunik 20240103
+    solar: 2022 # TODO check
+    battery: 2022 # TODO check
+    platform: 2022 # TODO ask Nick and Charlie
+    electrical_export_system: 2022 # also from ORBIT, so match wind assumptions. TODO ask Sophie Bradenkamp
+    desal: 2013 # from code citation: https://www.nrel.gov/docs/fy16osti/66073.pdf
+    electrolyzer: 2020 # 2020 for singlitico2021, 2016 # for simple h2 cost model in hopp (see https://www.hydrogen.energy.gov/pdfs/19009_h2_production_cost_pem_electrolysis_2019.pdf) ## 2020 # based on IRENA report https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Dec/IRENA_Green_hydrogen_cost_2020.pdf
+    h2_transport_compressor: 2016 # listed in code header
+    h2_storage:
+      pressure_vessel: 2022 # based on readme for Compressed_gas_function
+      pipe: 2019 # Papadias 2021
+      salt_cavern: 2019 # Papadias 2021
+      turbine: 2003 # assumed based on Kottenstette 2004
+      none: 2022 # arbitrary
+    h2_pipe_array: 2018 # ANL costs
+    h2_transport_pipeline: 2018 # same model for costs as the h2_pipe_array
+  wind:
+    expected_plant_cost: 'none'
+    
+electrolyzer:
+  rating: 180 # MW # 0.9*Plant rating appears near-optimal for 400 MW wind plant with 3 days of underground pipe storage # MW
+  electrolyzer_capex: 700 # $/kW conservative 2025 centralized. high 700, low 300 # based on https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Dec/IRENA_Green_hydrogen_cost_2020.pdf
+  time_between_replacement: 62320 # 62320 based on PEM Electrolysis H2A Production Case Study Documentation estimate of 7 years, 40000 based on current est (see unpublished report), 80000 hours based on OSW atb_year = 2025
+  replacement_cost_percent: 0.15 # percent of capex - H2A default case
+  cost_model: "singlitico2021" # "basic" is a basic cost model based on H2a and HFTO program record for PEM electrolysis. "singlitico2021" uses cost estimates from that paper
+  cluster_rating_MW: 40
```

Comment:
lines 53-61 we may want to use the electrolyzer.py input yaml, or at least match the names so conversion between the two is easy.

Reply:
some of these capabilities that these inputs are used for don't exist in the electrolyzer repo (yet - this will all be fixed with BERT)